src/index.html

```diff
@@ -15,18 +15,21 @@
       rel="stylesheet"
       href="style.css"
     />
+
   </head>
   <body>
     <input
       type="text"
       data-qa="keypress"
       placeholder="Try “Los Angeles“"
+      class="input input_item-1"
     />
-
+    <br>
```

Comment:
The `<br>` tag should be self-closing like `<br />` to maintain consistency with the XHTML style used in the rest of the document.